_id/1/step1.md

```diff
@@ -0,0 +1,11 @@
+Kita telah memasang minikube untuk kami. Periksa apakah terpasang dengan baik, dengan menjalankan perintah *minikube version*:
```

Comment:
this sentence doesn't make sense, how about removing "untuk kami"?
```suggestion
Kita telah melakukan pemasangan minikube. Periksa apakah terpasang dengan baik, dengan menjalankan perintah *minikube version*:
```

Reply:
oh ya, pardon, I missed this part 🙏